runtime/ftplugin/query.lua

```diff
@@ -2,5 +2,29 @@
 -- Language:	Tree-sitter query
 -- Last Change:	2022 Mar 29
 
+if vim.b.did_ftplugin == 1 then
+  return
+end
+
+-- Do not set vim.b.did_ftplugin = 1 to allow loading of ftplugin/lisp.vim
+
+-- use treesitter over syntax
+vim.treesitter.start()
+
+-- query linter
+local buf = vim.api.nvim_get_current_buf()
+local query_lint_on = vim.b.lint_on or { 'InsertLeave', 'CursorHoldI', 'BufEnter' }
+
+if query_lint_on and #query_lint_on > 0 then
+  vim.api.nvim_create_autocmd(query_lint_on, {
+    group = vim.api.nvim_create_augroup('querylint', {}),
```

Comment:
This'll cause autocmds to be reset when this ftplugin runs subsequent times. This includes when `vim.filetype.get_option()` runs.

Reply:
How should I change it? nvim_create_augroup spells: `  Create or get an autocommand group |autocmd-groups|.` Do you want other options like `{ clear = false }`?

Reply:
Do we even need a group?

Reply:
For now just do `{ clear = false }`.